projects/jupyter-server-ydoc/jupyter_server_ydoc/stores.py

```diff
@@ -30,3 +30,9 @@ class SQLiteYStore(LoggingConfigurable, _SQLiteYStore, metaclass=SQLiteYStoreMet
         help="""The document time-to-live in seconds. Defaults to None (document history is never
         cleared).""",
     )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Class is instantiated later, so we need to set the document_ttl here
+        self.document_ttl = int(self.document_ttl) if self.document_ttl is not None else None
```

Comment:
I don't understand what this changes. Could you write a test that shows that this fixes something?

Reply:
I added a test that checks whether the store has the correct parameter from the command line arguments. 

Without the changeset above the store writing fails immediately if document_ttl is set.